.github/workflows/ci.yml

```diff
@@ -0,0 +1,20 @@
+name: CI
+on:
+  pull_request:
+  push:
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    strategy:
+      fail-fast: false
+      matrix:
+        java: [8, 11]
+    steps:
+      - uses: actions/checkout@v1
+      - uses: olafurpg/setup-scala@v10
+        with:
+          java-version: "adopt@1.${{ matrix.java }}"
+      - uses: coursier/cache-action@v5
+      - run: sbt -v "+ test" checkScalariform
+      - run: rm -rf "$HOME/.ivy2/local"
```

Comment:
Could you elaborate why is this step necessary?

Reply:
sorry, this wasn't necessary 🙏 
It has been deleted 👍